stdlib/jdbc/src/main/ballerina/src/java.jdbc/Module.md

```diff
@@ -4,7 +4,29 @@ This module provides the functionality required to access and manipulate data st
 that is accessible via Java Database Connectivity (JDBC).
 
 **Prerequisite:** Please add the JDBC driver corresponding to the database you are trying to interact with, 
-as a native library dependency in your Ballerina project.
+as a native library dependency in your Ballerina project. Then, once you build the project with `ballerina build`
+command, you should be able run the resultant jar with,`java -jar` command.
+
+e.g. Ballerina.toml content
+
+```toml
+[project]
+org-name= "sample"
+version= "0.1.0"
+
+[platform]
+target = "java"
+
+    [[platform.libraries]]
+    artafactId = "mysql-connector-java"
+    version = "8.0.17"
+    path = "/path/to/mysql-connector-java-8.0.17.jar"
```

Comment:
We also need to say to set the path to the jar

Reply:
Added a sentence indicating that user needs to change the path appropriately.